lib/tasks/payment_breakdown.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :payment_calculation do
+  desc "run payment calculator for a given lead provider"
+  task breakdown: :environment do
+    logger = Logger.new($stdout)
+    logger.formatter = proc do |_severity, _datetime, _progname, msg|
+      "#{msg}\n"
+    end
```

Comment:
The more I see loggers and puts all over the place the more I wonder if we should have a common way of doing them. Oh well, something for later.